solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -355,6 +271,20 @@ const fetchOperatorsOf = async (web3Context, yourAddress) => {
     })
   }
 
+  const copiedOperatorsFromLiquidTokens = await getCopiedOperatorsFromLiquidTokens(
+    defaultAccount,
+    Array.from(operators.keys())
+  )
+  for (let i = 0; i < copiedOperatorsFromLiquidTokens.length; i++) {
+    operators.set(
+      web3Utils.toChecksumAddress(copiedOperatorsFromLiquidTokens[i]),
+      {
+        managedGrantInfo: {},
+        isWithdrawableForOperator: true,
```

Comment:
should be probably `false`, since contract will block the transaction anyway

Reply:
pls see 78469b8